pennylane/beta/queuing/measure.py

```diff
@@ -86,9 +194,6 @@ def circuit(x):
     Args:
         op (Observable): a quantum observable object
 
-    Returns:
-        MeasurementProcess: measurement process representing the measurement
```

Comment:
Do we still return a `MeasurementProcess` but just don't want to highlight?

Reply:
Yep